src/DssSpell.sol

```diff
@@ -28,6 +47,22 @@ contract DssSpellAction is DssAction {
         return false;
     }
 
+    address immutable internal OPTIMISM_GOV_RELAY = DssExecLib.getChangelogAddress("OPTIMISM_GOV_RELAY");
+    address immutable internal ARBITRUM_GOV_RELAY = DssExecLib.getChangelogAddress("ARBITRUM_GOV_RELAY");
+    address immutable internal STARKNET_GOV_RELAY = DssExecLib.getChangelogAddress("STARKNET_GOV_RELAY");
+
+    address constant internal OPTIMISM_L2_SPELL = 0xC077Eb64285b40C86B40769e99Eb1E61d682a6B4;
+    address constant internal ARBITRUM_L2_SPELL = 0x11dc6ed4c08da38b36709a6c8dbaac0eaedd48ca;
+    // address constant internal STARKNET_L2_SPELL =
+
+    uint256 public constant OPT_MAX_GAS = 100_000;
+
+    // run ./scripts/get-arb-relay-cost to generate the following 3 constants
```

Comment:
Between the time of someone running this script and the actual spell cast on mainnet there are at least the current 2 days of governance delay.
What would happen if the costs change in the meantime?
Is there a mechanism to retry with more gas? (if not maybe we can create such).